plugins/node.d/mysql_.in

```diff
@@ -698,7 +752,126 @@ $graphs{tmp_tables} = {
     ],
 };
 
+#---------------------------------------------------------------------
+#  Plugin Graphs
+#  These are mysql plugins of type INFORMATION SCHEMA
+#
+#  These will be added to $graphs if available
+#---------------------------------------------------------------------
 
+my %graph_plugins = ();
+
+$graph_plugins{query_response_time} = {
+    count => {
+        config => {
+            global_attrs => {
+                title  => 'Query Response Time Count',
+                vlabel  => 'queries per ${graph_period}',
+            },
+            data_source_attrs => {
+                draw  => 'LINE2',
+                type => 'DERIVE',
+            },
+        },
+        # data_sources are populated by sub plugin_query_response_time
+        data_sources => [
+        ],
+   },
+   total => {
+        config => {
+            global_attrs => {
+                title  => 'Query Response Time Total',
+                vlabel  => 'query time (microseconds) per ${graph_period}',
```

Comment:
According to the ticket this graph has been broken forever? If so the change is required.

But I would possibly change the name of the time series to ensure that there is no confusion about the scale of the values and a break in the graph. 

BUT! As a basic principle plugins should report things in base units: bytes, bits, seconds, and such and never megabytes, gigabits or microseconds. Otherwise we can easily run in to kilo-microseconds.  Using the base unit allows rrd to scale the units according to SI. Therefore I would recommend to keep this in seconds.  Since it needs to be sent from the plugin in microseconds a CDEF to compute seconds is  in order.

Reply:
Indeed - this is a bit of confusing case!

> According to the ticket this graph has been broken forever?

As far as I understand the git history, the graph was added to the plugin to `master` in 2015. For `stable-2.0` it would be introduced with the commit above.

Thus we should have all the freedom for changes, I guess.

I prepared #935 for the cdef you suggested. Do you have any comments on that?